scene/gui/control.cpp

```diff
@@ -3267,11 +3265,8 @@ void Control::_notification(int p_notification) {
 				if (get_viewport() != nullptr) {
 					get_viewport()->_gui_hide_control(this);
 				}
-			} else {
-				data.minimum_size_valid = false;
-				_update_minimum_size();
-				_size_changed();
 			}
+			_update_minimum_size();
```

Comment:
Is there any point in updating minimum size if the Control is invisible?

Reply:
Let the parent container recalculate the size instead of continuing to use the cached minimum size.

Reply:
But containers ignore invisible children.

Reply:
Yes, a change in visibility of a child control will notify the parent container to update the minimum size. Not necessary here.